### DISCUSSION TITLE
Game crash on startup MacOS

### DISCUSSION THREAD

**Discussion by ini1911** [voxeltycoon/issues#639] [Troubleshooting]
**Describe the bug**

The Game crash on startup. Just a short black blink, then the MacOS errorhandler came up.
Gatekeeper is disabled.

**Steps to reproduce**
Just start the Game via Steam

**Build version**
?

**Platform**
MacOS 10.13..6
2,5 GHz Intel Core i5
12 GB 1067 MHz DDR3
AMD Radeon HD 6750M 512 MB

**Reproducibility**
How often an issue reproduces: every time.

**Logs/saves**
[Player.log](https://github.com/voxeltycoon/bug-tracker/files/6388112/Player.log)

**Comment by dddccx** [comment]
ini1911, your computer doesn't meet the basic requirements to even run the game, you need at least double the ram you have now to have the game function with basic graphics. 1000% recommend you get a computer that has much more ram and a better gpu because what you have now could barely function using even the internet, let alone gaming.

**Reply by andrewpey** [reply]
Actually, 12 GB of RAM looks pretty fine for the game.

**Comment by ini1911** [comment]
i know, my Mac is a old boy, but the System requirements a fullfilled...

**Comment by andrewpey** [comment]
Please try the following:
1. Open game properties on Steam
2. Paste `-force-opengl` text to **Launch options** text field
3. Run the game and see if this helps

**Comment by ini1911** [comment]
ive tryed the command, but didnt work out.
